docs/user_manual/calculations.md

```diff
@@ -653,6 +654,22 @@ Internally, to achieve an optimal regulated tap position, the control algorithm
 | {py:class}`TapChangingStrategy.min_voltage_tap <power_grid_model.enum.TapChangingStrategy.min_voltage_tap>` | `tap_max`            | step up                | Find the tap position that gives the lowest control side voltage within the `u_band`  |
 | {py:class}`TapChangingStrategy.max_voltage_tap <power_grid_model.enum.TapChangingStrategy.max_voltage_tap>` | `tap_min`            | step down              | Find the tap position that gives the highest control side voltage within the `u_band` |
 
+##### Search methods used for tap changing optimization
+
+Given the discrete nature of the finite tap ranges, we use the following two search methods to find the next tap position along the exploitation direction.
```

Comment:
```suggestion
Given the discrete nature of the finite tap ranges, we use the following search methods to find the next tap position along the exploitation direction.
```
not a real issue but prevents divergence in the future

Reply:
addressed in the latest commit.